q2_chemistree/_hierarchy.py

```diff
@@ -7,51 +7,72 @@
 # ----------------------------------------------------------------------------
 
 import biom
+import pandas as pd
 from sklearn.metrics import pairwise_distances
 from scipy.spatial.distance import squareform
 from scipy.cluster.hierarchy import linkage
 from skbio import TreeNode
 
+from ._collate_fingerprint import collate_fingerprint
+from ._match import match_label
+from ._semantics import CSIDirFmt
 
-def make_hierarchy(collated_fingerprints: biom.Table,
-                   prob_threshold: float = None,
-                   distance_metric: str = 'jaccard') -> TreeNode:
+
+def build_tree(relabeled_fingerprints: pd.DataFrame) -> TreeNode:
     '''
     This function makes a tree of relatedness between mass-spectrometry
-    features using molecular substructure information.
+    features using molecular substructure fingerprints.
+    '''
+    distmat = pairwise_distances(X=relabeled_fingerprints,
+                                 Y=None, metric='jaccard')
```

Comment:
Would it be helpful for the metric to be parameterized?

Reply:
We compared some metrics and we didn't observe any big differences hence why we decided to leave jaccard for now.